assets/csidriveroperators/openstack-cinder/standalone/deployment.patch.yaml

```diff
@@ -13,3 +13,15 @@ spec:
         - key: node-role.kubernetes.io/master
           operator: Exists
           effect: "NoSchedule"
+      volumes:
+        # This is the legacy location of the CA cert. The cert is now provided in
+        # the credentials secret generated by cloud-credential-operator, so this
+        # will not be used outside of an upgrade scenario
+        # TODO(stephenfin): Remove in 4.20
+        - name: legacy-cacert
+          configMap:
+            name: cloud-provider-config
```

Comment:
what is the difference between `cloud-provider-config` and `openstack-cloud-credentials` ?

Reply:
`cloud-provider-config` is the name used by the Installer. `openstack-cloud-config` is the name used by Hypershift. This move was done as part of https://github.com/openshift/cluster-storage-operator/pull/557/commits/a88aa28ea3da635469404449275e534b517e5798. I've done it because (repeating from the commit message):

> As noted inline [ed: in the commit], the definition in the base asset was only used in the generated standalone asset, so it actually belongs in the standalone patch. This has no impact on the generated assets.

Reply:
Yes, a comment inline will be nice.

Reply:
Sorry, I meant I would leave a comment on the note, which is [here](https://github.com/openshift/cluster-storage-operator/pull/557#discussion_r2022886055).

Do you want another one note here?